.github/workflows/sync-docs-from-cli-repo.yml

```diff
@@ -17,7 +17,8 @@ jobs:
   config-sync:
     name: Sync docs to docs site repo
     runs-on: ubuntu-latest
-
+    permissions:
+      contents: read
```

Comment:
📣 Checking out the repo is all that's required with the default token!

An application token helps with other permissions. Testing after the changes of #52-